lib/incito-browser/incito.js

```diff
@@ -1,308 +1,614 @@
+                    if (item.span?.name != null) {
+                        const spanName = item.span.name;
+
+                        return `<span data-name="${spanName}">${escapedText}</span>`;
+                    }
+                    return escapedText;
+                });
+                text = text.join('');
             } else {
-                // make sure visibleRendered gets triggered even
-                // if renderedOutsideOfViewport wasn't
-                this.renderedOutsideOfViewport = true;
-                this.trigger('allRendered');
+                text = escapeHTML(text);
+            }
+
+            if (view.text_prevent_widow) {
+                text = text
+                    .replace(/&nbsp;([^\s]+)$/, ' $1')
+                    .replace(/\s([^\s]+)\s*$/, '&nbsp;$1');
             }
 
-            if (this.renderedOutsideOfViewport && !triggeredVisibleRendered) {
-                this.trigger('visibleRendered');
+            contents = text.replace(/\n/g, '<br>');
 
-                triggeredVisibleRendered = true;
+            if (
+                Array.isArray(view.font_family) &&
+                view.font_family.length > 0
+            ) {
+                styles['font-family'] = view.font_family.join(', ');
             }
 
-            if (this.renderedOutsideOfViewport) {
-                this.lazyload(0);
+            if (view.text_size != null) {
+                styles['font-size'] = `${view.text_size}px`;
             }
-        };
 
-        this.el.className = 'incito';
-        if (this.incito.locale != null) {
-            this.el.setAttribute('lang', this.incito.locale);
-        }
+            if (view.line_spacing_multiplier != null) {
+                styles['line-height'] = view.line_spacing_multiplier;
+            }
 
-        loadFonts(this.incito.font_assets);
-        this.applyTheme(this.incito.theme);
+            if (view.text_color != null) {
+                styles.color = view.text_color;
+            }
 
-        this.containerEl.appendChild(this.el);
+            if (textStyles.indexOf('bold') !== -1) {
+                styles['font-weight'] = 'bold';
+            }
 
-        // do first render synchronously unless we're very lazy
-        if (this.renderLaziness === 2) {
-            this.renderCallbackHandle = requestIdleCallback(render);
-        } else {
-            syncIdleCallback(render);
-        }
+            if (textStyles.indexOf('italic') !== -1) {
+                styles['font-style'] = 'italic';
+            }
 
-        document.addEventListener('scroll', this.lazyloader, true);
-        window.addEventListener('resize', this.lazyloader, false);
+            if (Array.isArray(view.text_decoration_line)) {
+                styles['text-decoration-line'] = view.text_decoration_line.join(
+                    ' '
+                );
+            }
 
-        return this;
-    }
+            const textShadow = getTextShadow(view);
 
-    destroy() {
```

Comment:
`IncitoPublicationKit`'s `Viewer` calls this method for cleanup here https://github.com/shopgun/shopgun-js-sdk/blob/develop/lib/kits/incito-publication/viewer.js